makiflow/tools/test_visualizer.py

```diff
@@ -42,22 +42,24 @@ def plot_test_values(test_values, legends, x_label, y_label, save_path=None):
         plt.close(fig)
 
     @staticmethod
-    def plot_numpy_dist_obs(values, legends, save_path=None, axes_x=None, axes_y=None):
+    def plot_numpy_dist_obs(values, legends, save_path=None, x_axis=None, y_axis=None):
         """
+        Flexibly plot a univariate distribution of observations.
+
         Parameters
         ----------
         values : list
-            Set of values that need to plot.
+            Set of numpy.ndarray values that need to plot.
```

Comment:
ndarrays of what shape? If the array can be of arbitrary shape, then state it explicitly. 
What does it actually mean to plot in this case? It could be formulated as "... of values which distributions to plot". Saying explicitly what actually will be plotted makes everything much more clear.